### DISCUSSION TITLE
where is the issues? We had lots of issues in old repo

### DISCUSSION THREAD

**Discussion by braindevices** [KDE/ghostwriter#825] [Q&A]
I just realized the original project seems transferred to KDE. However, I cannot find the issues any more. We had lots of issues in https://github.com/wereturtle/ghostwriter/issues. I cannot even find my own issue any more. Do you plan to port the issues to gitlab? I only see 1 single issue in gitlab repo. So what the hell is going on here?

Where should I report new issue?

**Comment by wereturtle** [comment]
Sorry, someone closed them for me.  I'll be sure to transfer them.  Right now everything is hectic and crazy getting ready for the formal KDE review process.  Please be patient.

> So what the hell is going on here?

*Sigh*
